api/test/unit/utils/test_image_proxy.py

```diff
@@ -47,6 +49,12 @@
 </svg>"""
 
 
+cache_availability_params = pytest.mark.parametrize(
+    "is_cache_reachable, cache_name",
+    [(True, "redis"), (False, "unreachable_redis")],
+)
```

Comment:
Is there anyway to reuse this, even if it means importing it from somewhere? That way the description I mentioned above only has to be made once potentially.

Reply:
They're all different in terms of the fixtures they use so defining them in one place would not be very helpful.